code/modules/mob/living/carbon/human/species.dm

```diff
@@ -295,8 +295,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			A.cure(FALSE)
 
 	SEND_SIGNAL(C, COMSIG_SPECIES_GAIN, src, old_species)
-
-	C.lewd = C.client.prefs.lewdchem
```

Comment:
Please don't follow the below snowflakes and place this mere client preference outside of species.dm.
And if possible, remove the lewd var in favour of a less quirkier code and replace all its istances with `C.client?.prefs.lewdchem`.

Reply:
Ah, yeah, makes sense

Reply:
I did this for all instances, hopefully there's no oversight with this. Testing it seems to show that it's working.